usr/share/rear/restore/NBKDC/default/40_restore_backup.sh

```diff
@@ -0,0 +1,16 @@
+LogPrint "
+
+The System is now ready for restore. Please start the restore task from the NovaBACKUP DataCenter Central Management!
+
+! Remember that the restore target must be set to '/mnt/local' !
+
+For further documentation see the following link:
+
+ http://www.novastor.com/help-html/dc/en-US/index.html
+
+
+after the restore has finished quit this wizard with the command 'exit' to continue.
+"
+
+rear_shell "Did you restore the backup to /mnt/local ? Are you ready to continue recovery ?"
```

Comment:
@andyvb070 why do you escape to the rear shell? The user needs to exit the shell again to continue the recovery which I find very confusing. The rear shell was introduced for debugging purposes. I would rather keep it for these kind of exercises and to use it for recovery purposes.